static/app/components/events/interfaces/nativeFrame.tsx

```diff
@@ -423,14 +455,19 @@ const FileName = styled('span')`
 const RowHeader = styled('span')<{
   expandable: boolean;
   expanded: boolean;
+  isSubFrame: boolean;
   stacktraceChangesEnabled: boolean;
 }>`
   display: grid;
-  grid-template-columns: repeat(2, auto) 1fr repeat(2, auto) ${space(2)};
+  grid-template-columns:
+    repeat(2, auto) 1fr repeat(${p => (p.stacktraceChangesEnabled ? 3 : 2)}, auto)
+    ${space(2)};
   grid-template-rows: repeat(2, auto);
   align-items: center;
   align-content: center;
   column-gap: ${space(1)};
+  background: ${p =>
+    p.stacktraceChangesEnabled && p.isSubFrame ? `${p.theme.surface100} !important` : ''};
```

Comment:
Can we remove this !important? We want to avoid those if at all possible.

This also seems to conflict with the `background-color` below. Can you add this logic to that instead?